java/arcs/core/host/EntityHandleManager.kt

```diff
@@ -224,32 +209,32 @@ class WriteSingletonHandleAdapter<T : Entity>(
 ) : BaseHandleAdapter(name, storageProxy),
     WriteSingletonHandle<T>,
     WriteSingletonOperations<T> by
-        WriteSingletonOperationsImpl<T>(name, storageProxy, entityPreparer)
+    WriteSingletonOperationsImpl<T>(name, storageProxy, entityPreparer)
```

Comment:
What's the case for keeping this at the same indentation level? I liked the indentation, it helped see that this is a delegation, not another interface being implemented.

Reply:
IntelliJ doing its thing... I've reverted it.

Reply:
Actually, I think this should probably be the one-arg-per-line thing.